assets/css/abstracts/_mixins.scss

```diff
@@ -118,3 +118,7 @@
 	// IE11 doesn't support overflow-wrap neither word-break: break-word, so we fallback to -ms-work-break: break-all.
 	-ms-word-break: break-all;
 }
+
+@function rem($size, $base: 16px ) {
```

Comment:
I noticed `font-size()` and `rem()` mixins have different signatures (`font-size` requires unitless variables while `rem` requires `px` variables). I think it would be great to unify them, but I didn't want to cause conflicts with #2320, instead I think we can deal with this in a follow-up.

Reply:
passing units or not shouldn’t be an issue as long as the base is in the same unit-less world, using px made sense to me since we're converting from px.